### PR TITLE
fix(CI): Fix Lighthouse report generation

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This fixes the Lighthouse report generation and upgrades Lighthouse to 10.0.0: https://developer.chrome.com/blog/lighthouse-10-0/

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [x] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: .github/actions/lighthouse-performance-testing/action.yml [hunk 1]
```diff
@@ -19,9 +19,53 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Install dependencies
```
Comment (bdauria) [inline .github/actions/lighthouse-performance-testing/action.yml:22]: @Thebarda You might want to use: https://github.com/shivammathur/setup-php

Comment (Thebarda) [reply .github/actions/lighthouse-performance-testing/action.yml]: wow

File: .github/actions/lighthouse-performance-testing/action.yml [hunk 2]
```diff
@@ -19,9 +19,38 @@ inputs:
...
+        coverage: none
+        tools: composer:v2
+
     - name: "Start Centreon container"
       run: docker run -p 4000:80 -d -v "/var/run/docker.sock:/var/run/docker.sock" --name lighthouse-tests-${{ inputs.module }} ${{ inputs.image }}:${{ inputs.image_version }}
       shell: bash
-    - name: "Run Lighthouse"
-      run: docker run --shm-size 2gb -e MODULE_PATH=${{ inputs.path }} -i -v "/var/run/docker.sock:/var/run/docker.sock" -v "$PWD:/tmp" -w "/tmp" --net="host" --entrypoint "/tmp/.github/scripts/lighthouse-performance-testing.sh" centreon.jfrog.io/docker/lighthouse:${{ inputs.image_lighthouse_version }}
+
+    - name: Set up and running centreon-injector
```
Comment (bdauria) [inline .github/actions/lighthouse-performance-testing/action.yml:34]:
```suggestion
    - name: Setup and run centreon-injector
```

**Review by wtermellil — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Review by kduret — CHANGES_REQUESTED** [review]
remove `composer-setup.php` file

**Review by kduret — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Comment by tuntoja** [comment]
@tom-rt when this will be merged, you may want to re enable Lighthouse jobs in pipelines where it was disabled (modules and centreon)

**Pull request merged**